wolfcrypt/src/aes.c

```diff
@@ -27,9 +27,10 @@
 #include <wolfssl/wolfcrypt/settings.h>
 
 #ifndef NO_AES
-
 #include <wolfssl/wolfcrypt/aes.h>
 
+#ifndef WOLFSSL_ARMASM
```

Comment:
There are portions of AES keywrap that are dependent on error-crypt.h and misc.c. With this current macro guard those header files do not get included in aes.c. "./configure --enable-all --enable-armasm && make"